gadgets/trace_malloc/gadget.yaml

```diff
@@ -0,0 +1,28 @@
+name: trace malloc
+description: use uprobe to trace malloc and free in libc.so
+tracers:
+  events:
```

Comment:
The name of the tracer needs to match what's defined in GADGET_TRACER():
s/events/malloc

Reply:
Thanks, will fix in the next PR.

Reply:
btw, if there a event field related to the `RUNTIME.CONTAINERNAME` column?

Reply:
The columns from container enrichment are automatically added when there is a `mntns_id` field.